### DISCUSSION TITLE
Website troubleshooting

### DISCUSSION THREAD

**Discussion by ghost** [community/community#137972] [Other Feature Feedback, Questions, & Ideas]
I created a github page and I uploaded all the necessary files (index.html, styles.css, scripts.js, ...), but when I visited the site after publishing it I found this statement "Coming soon!"

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @techTECNO89 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by ThisIs-Developer** [comment]
Hey there! things to check:

1. **Make sure your `index.html` file is in the right place** and named correctly. GitHub Pages is case-sensitive, so double-check that it's `index.html`, not `Index.html`.

2. **Verify your repository settings** to ensure the correct branch and root directory are selected for GitHub Pages.

If you're still seeing the "Coming soon!" message, check the build logs for any errors and make sure your repository is public. Keep at it, and let me know how it goes!

For more guidance, you can check [GitHub's documentation](https://docs.github.com/en/pages)

If this helps, please mark the answer as helpful!

**Comment by LiteBrite82** [comment]
that looks like a DNS configuration issue. Can you provide a screenshot of your GitHub pages settings and your DNS provider settings?
